great_expectations/rule_based_profiler/parameter_builder/parameter_container.py

```diff
@@ -140,21 +160,18 @@ def build_parameter_container(
     :param parameter_values
     Example of required structure for "parameter_values" (matching the type hint in the method signature):
     {
-        "$parameter.date_strings.tolerances.max_abs_error_time_milliseconds": {
-            "value": 100, The key must be "value"; the actual value can be of Any type.
-            "details": {  # This dictionary is Optional; however, if present, the key to it must be "details".
-                "max_abs_error_time_milliseconds": {
-                    "confidence": {  # Arbitrary dictionary key
-                        "success_ratio": 1.0,  # Arbitrary entries
-                        "comment": "matched template",  # Arbitrary entries
-                    }
-                },
+        "$parameter.date_strings.tolerances.max_abs_error_time_milliseconds.val": 100, The actual value can of Any type.
+        # The metadata (details) dictionary is Optional.
+        "$parameter.date_strings.tolerances.max_abs_error_time_milliseconds.metadata": {
+            "max_abs_error_time_milliseconds": {
+                "confidence": {  # Arbitrary dictionary key
+                    "success_ratio": 1.0,  # Arbitrary entries
+                    "comment": "matched template",  # Arbitrary entries
+                }
             },
         },
-        "$parameter.tolerances.mostly": {
-            "value": 9.0e-1
-            "details": None  # The "details" key is entirely Optional (it is exclusively for informational purposes).
-        },
+        # While highly recommended, the use of ".val" and ".metadata" keys is conventional (it is not enforced).
```

Comment:
This seems to contradict L161

Reply:
@talagluck With the updated L161 comment, the lack of contradiction should be more apparent now.  Please let me know if this can be explained better and I will incorporate.